.github/workflows/operate-first-publish.yaml

```diff
@@ -8,42 +8,43 @@ on:
 
 jobs:
   build:
-    name: Update Operate First directory
+    name: Create PR in operate-first/apps repo with updated manifests
     runs-on: ubuntu-latest
     steps:
-    
-      - name: Checkout Operate First repository
-        uses: actions/checkout@v2
-        with:
-          repository: fybrik/operate-first-apps
-          path: operate-first-repo
       - name: Checkout Fybrik repository
         uses: actions/checkout@v2
         with:
           repository: fybrik/fybrik
           path: fybrik-repo
+      - name: Checkout original Operate First repository
+        uses: actions/checkout@v2
+        with:
+          repository: operate-first/apps
+          path: operate-first-repo
       - run: |
-          rm -r fybrik-repo/samples/operate-first/*/
-          rm fybrik-repo/samples/operate-first/*.yaml
+          rm -r fybrik-repo/integration/operate-first/*/
```

Comment:
I don't think that we need the generated files to be committed at all (and then you won't need to delete them here)

Reply:
@roee88 I removed the generated files and this line